src/js/config.js

```diff
@@ -111,17 +111,17 @@ export const defaultI18n = {
       enableOtherMsg: 'Let users to enter an unlisted option',
       fieldNonEditable: 'This field cannot be edited.',
       fieldRemoveWarning: 'Are you sure you want to remove this field?',
-      fileUpload: 'File Upload',
+      fileUpload: 'File Upload',         
```

Comment:
lots of white space being added

Reply:
Yeah I tried to fix all of these in the last update.